docker/config/grafana/provisioning/dashboards/lido_basset/lido_basset.json

```diff
@@ -75,6 +75,131 @@
         "x": 0,
         "y": 0
       },
+      "id": 6,
+      "options": {
+        "legend": {
+          "calcs": [],
+          "displayMode": "list",
+          "placement": "bottom"
+        },
+        "tooltip": {
+          "mode": "single"
+        }
+      },
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "bluna_reward_config_crc32{}",
+          "interval": "",
+          "legendFormat": "",
+          "refId": "A"
+        },
+        {
+          "exemplar": true,
+          "expr": "hub_config_crc32{}",
+          "hide": false,
+          "interval": "",
+          "legendFormat": "",
+          "refId": "B"
+        },
+        {
+          "exemplar": true,
+          "expr": "hub_parameters_crc32{}",
+          "hide": false,
+          "interval": "",
+          "legendFormat": "",
+          "refId": "C"
+        },
+        {
+          "exemplar": true,
+          "expr": "reward_dispatcher_config_crc32{}",
+          "hide": false,
+          "interval": "",
+          "legendFormat": "",
+          "refId": "D"
+        },
+        {
+          "exemplar": true,
+          "expr": "validators_registry_config_crc32{}",
+          "hide": false,
+          "interval": "",
+          "legendFormat": "",
+          "refId": "E"
+        },
+        {
+          "hide": false,
+          "refId": "F"
+        },
+        {
+          "exemplar": true,
+          "expr": "airdrop_registry_config_crc32{}",
+          "hide": false,
+          "interval": "",
+          "legendFormat": "",
```

Comment:
Please add legends. And something must be done with the layout:
<img width="827" alt="image" src="https://user-images.githubusercontent.com/6124982/127980448-fc21cc62-f30a-4210-be17-4e31a9dba543.png">

Reply:
added legends and layout configured as 3 columns